### PR TITLE
Update README.md

### LINKED ISSUE #106: Update instructions for publishing to `Objecitonary/home`
Amend corresponding `README` section for better clarity on publishing steps.

### PR DESCRIPTION
**Pull request by mximp**
Closes #106

### CODE REVIEW

**Comment by mximp** [comment]
@yegor256 please review and merge.

File: README.md [hunk 1]
```diff
@@ -61,6 +61,15 @@ will be updated. All users will be able to use your objects.
 
 ## How to Publish a Library
 
+Once the library is ready for publishing (i.e. all required changes are released)
+it can be published.
+
+### Pull recent changes from this repo
```
Comment (yegor256) [inline README.md:67]: @mximp don't use headings when the block of text inside the "section" consists of a single sentence. Headings are needed in order to separate more or less big blocks of text from each other. Here, without headings the text would look much more readable.

Comment (mximp) [reply README.md]: fixed

**Comment by mximp** [comment]
@yegor256 please check this option

**Comment by yegor256** [comment]
@mximp what is the point of giving titles to paragraphs? :) When you tell a story to someone about how to, say, cook an omelet. Do you start every sentence with a title? :) Just tell the story, don't label paragraphs.

**Comment by mximp** [comment]
> @mximp what is the point of giving titles to paragraphs? :) When you tell a story to someone about how to, say, cook an omelet. Do you start every sentence with a title? :) Just tell the story, don't label paragraphs.

@yegor256 The idea is that instructions are easer to read when have clear steps. It's more like giving the algorithm rather than telling a story.

I've removed the titles though.

**Comment by yegor256** [comment]
@mximp thanks!

**Pull request merged**